spec/jekyll_seo_tag_spec.rb

```diff
@@ -247,6 +247,16 @@
           expect(output).to match(expected)
         end
       end
+
+      context 'with page.image' do
+        let(:site) { make_site('twitter' => site_twitter, 'url' => 'http://example.invalid') }
+        let(:page) { make_page('image' => 'foo.png') }
```

Comment:
Root. I think most folks would place images in /img/ or /assets/. But the config or code does not preclude either use. Would it be clearer if that kind of usage were in the tests?